core/src/ast/snapshots/diplomat_core__ast__methods__tests__nonstatic_methods-2.snap

```diff
@@ -1,6 +1,6 @@
 ---
 source: core/src/ast/methods.rs
-expression: "Method::from_syn(&syn::parse_quote! {\n                                  fn\n                                  foo(& mut self, x : u64, y : MyCustomStruct)\n                                  -> u64 { x }\n                              }).unwrap(),\n                 &Path::empty().sub_path(\"MyStructContainingMethod\".to_string()))"
+expression: "Method::from_syn(&syn::parse_quote! {\n                      fn foo(& mut self, x : u64, y : MyCustomStruct) -> u64\n                      { x }\n                  },\n                 &Path::empty().sub_path(\"MyStructContainingMethod\".to_string()))"
```

Comment:
Question: I don't see where in the code you changed anything that would cause the deletion of the `.unwrap()`.  Seems like a good change though.

Reply:
Oh, I suspect insta has some fuzzy-matching for matching tests, so if you update a test then it might not need the `expression` updated (since `expression` is just used for matching which test was called), but if the result changes then it will get replaced